Makefile

```diff
@@ -1,139 +1,155 @@
-# Application
-APP_ROOT := $(CURDIR)
+# Configuration
+APP_ROOT := $(abspath $(lastword $(MAKEFILE_LIST))/..)
 APP_NAME := emu
 
-# Anaconda
-CONDA := $(shell command -v conda 2> /dev/null)
-ANACONDA_HOME := $(shell conda info --base 2> /dev/null)
-CONDA_ENV := $(APP_NAME)
-
-TEMP_FILES := *.egg-info *.log *.sqlite
+# Bumpversion 'dry' config
+# if 'dry' is specified as target, any bumpversion call using 'BUMP_XARGS' will not apply changes
+BUMP_XARGS ?= --verbose --allow-dirty
+ifeq ($(filter dry, $(MAKECMDGOALS)), dry)
+	BUMP_XARGS := $(BUMP_XARGS) --dry-run
+endif
+.PHONY: dry
+dry: setup.cfg
+	@-echo > /dev/null
 
 # end of configuration
 
-.DEFAULT_GOAL := all
+.DEFAULT_GOAL := help
 
 .PHONY: all
 all: help
 
 .PHONY: help
 help:
-	@echo "Please use \`make <target>' where <target> is one of"
-	@echo "  help        to print this help message. (Default)"
-	@echo "  install     to install $(APP_NAME) by running 'python setup.py develop'."
-	@echo "  start       to start $(APP_NAME) service as daemon (background process)."
-	@echo "  stop        to stop $(APP_NAME) service."
-	@echo "  restart     to restart $(APP_NAME) service."
-	@echo "  status      to show status of $(APP_NAME) service."
-	@echo "  clean       to remove *all* files that are not controlled by 'git'. WARNING: use it *only* if you know what you do!"
+	@echo "Please use 'make <target>' where <target> is one of:"
+	@echo "  help              to print this help message. (Default)"
+	@echo "  install           to install app by running 'pip install -e .'"
+	@echo "  develop           to install with additional development requirements."
+	@echo "  start             to start $(APP_NAME) service as daemon (background process)."
+	@echo "  stop              to stop $(APP_NAME) service."
+	@echo "  restart           to restart $(APP_NAME) service."
+	@echo "  status            to show status of $(APP_NAME) service."
+	@echo "  clean             to remove all files generated by build and tests."
 	@echo "\nTesting targets:"
-	@echo "  test        to run tests (but skip long running tests)."
-	@echo "  testall     to run all tests (including long running tests)."
-	@echo "  pep8        to run pep8 code style checks."
+	@echo "  test              to run tests (but skip long running tests)."
+	@echo "  test-all          to run all tests (including long running tests)."
+	@echo "  lint              to run code style checks with flake8."
```

Comment:
This is a welcome change. Some other projects include Black formatting. This leaves an opening for the discussion.